sample_project/Assets/SampleViewer/Samples/Geocoding/Geocoder.cs

```diff
@@ -322,23 +322,8 @@ private async Task<string> SendLocationQuery(string location)
     /// <param name="markerGO"></param>
     void PlaceOnGround(GameObject markerGO)
     {
```

Comment:
Update the function summary (or remove the function since its only being called from one place)